src/main/java/use_cases/default_game/MazeInteractor.java

```diff
@@ -204,6 +208,9 @@ public synchronized void movePlayer(int keycode) {
         if (player.getStageClear()) {
             outputBoundary.changeState(IGamePanelOutputBoundary.LEVEL_CLEAR_STATE);
             outputBoundary.recordStamina(player.getStamina());
+
+            updateScore.updateScore(player.getStamina(), mazeLevel, "arifa");
```

Comment:
it shouldn't say "arifa"

Reply:
Yep! That was just to test if it was working with a specific user. I just pushed the changes that store the username after the user login/registers.